app/assets/stylesheets/govuk_publishing_components/components/govspeak/_media-player.scss

```diff
@@ -11,5 +11,12 @@
 
   .media-player {
     @include media-player;
+
+    // We need a specific styling on form elements otherwise it falls through
+    // to the browser default, this is needed in govuk_publishing_components
+    // as there is not a global reset like there is in govuk_frontend_toolkit
+    button {
+      @include core-16;
```

Comment:
can we use the `govuk-frontend` mixins instead? `@include govuk-font($size: 16);`